utils/node_local_test.py

```diff
@@ -2762,17 +2754,18 @@ def test_cont_clone(self):
 
         # Now create a container uuid and do an object based copy.
         # The daos command will create the target container on demand.
-        container = str(uuid.uuid4())
         cmd = ['container',
                'clone',
                '--src',
                'daos://{}/{}'.format(self.pool.uuid, self.container),
                '--dst',
-               'daos://{}/{}'.format(self.pool.uuid, container)]
+               'daos://{}/'.format(self.pool.uuid)]
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string